src/ontology/Makefile

```diff
@@ -814,13 +809,9 @@ ROBOT_RELEASE_IMPORT_MODE=$(ROBOT) merge --input $<
 # ROBOT pipeline that removes imports, then merges components. This is for release artefacts that start from "base"
 ROBOT_RELEASE_IMPORT_MODE_BASE=$(ROBOT) remove --input $< --select imports --trim false merge $(patsubst %, -i %, $(OTHER_SRC)) 
 
-# base: A version of the ontology that does not include any externally imported axioms.
-$(ONT)-base.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC) $(IMPORT_FILES)
```

Comment:
This is base-plus, since you reverted back to ODK 1.4.2 (which is a good idea), it may be we need to move this goal into the phenio.Makefile for the time being

Reply:
Great, it's there now.